### ISSUE TITLE
Cannot read property 'contextValue' of undefined

### ISSUE THREAD

**Issue by Konard** [microsoft/vscode-docker#2876]
Does this occur consistently? 
Repro steps:

1. I'm not sure how to reproduce it.

Action: vscode-docker.containers.remove
Error type: TypeError
Error Message: Cannot read property 'contextValue' of undefined

Version: 1.12.1
OS: linux
OS Release: 5.4.0-70-generic
Product: Visual Studio Code
Product Version: 1.55.1
Language: en

<details>
<summary>Call Stack</summary>

```
extension.bundle.js:2:4653971
Object.multiSelectNodes extension.bundle.js:2:4653954
removeContainer extension.bundle.js:2:4308782
extension.bundle.js:2:4339107
extension.bundle.js:2:5097459
Object.<anonymous> extension.bundle.js:2:5073332
extension.bundle.js:2:5072458
p extension.bundle.js:2:5072144
Object.callWithTelemetryAndErrorHandling extension.bundle.js:2:5073240
Object.<anonymous> extension.bundle.js:2:5097203
extension.bundle.js:2:5096802
p extension.bundle.js:2:5096488
extension.bundle.js:2:5097050
S._executeContributedCommand extensionHostProcess.js:86:106770
S.$executeContributedCommand extensionHostProcess.js:86:107208
u._doInvokeHandler extensionHostProcess.js:90:12794
u._invokeHandler extensionHostProcess.js:90:12478
u._receiveRequest extensionHostProcess.js:90:11145
u._receiveOneMessage extensionHostProcess.js:90:9931
extensionHostProcess.js:90:8032
_.fire extensionHostProcess.js:57:1836
E.fire extensionHostProcess.js:65:15496
extensionHostProcess.js:105:30083
_.fire extensionHostProcess.js:57:1836
E.fire extensionHostProcess.js:65:15496
t._receiveMessage extensionHostProcess.js:65:20754
extensionHostProcess.js:65:17640
_.fire extensionHostProcess.js:57:1836
acceptChunk extensionHostProcess.js:65:12861
extensionHostProcess.js:65:12209
Socket.w extensionHostProcess.js:105:13173
Socket.emit events.js:315:20
Socket.EventEmitter.emit domain.js:483:12
addChunk _stream_readable.js:295:12
readableAddChunk _stream_readable.js:271:9
Socket.Readable.push _stream_readable.js:212:10
Pipe.onStreamRead stream_base_commons.js:186:23
```

</details>

**Comment by bwateratmsft** [comment]
@karolz-ms I don't know for sure but if I had to take a guess, one of the tree items' object IDs changed inbetween this command starting and finishing. Subsequently the object became undefined.

We should look at the parts of the object ID...ideally they wouldn't be changing without, e.g., a container status change.

**Comment by bwateratmsft** [comment]
Probably fixed or mitigated by #3085 so closing for now.
